project/urls.py

```
@@ -22,6 +24,7 @@
     path("category/<slug:slug>/", category_courses, name="category_courses"),
     path("payment/", include("mindjunkies.payments.urls")),
     path("forums/", include("mindjunkies.forums.urls")),
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)